index.html

```diff
@@ -63,12 +63,219 @@ <h1 class="hero__title">Hey there. I’m Kyrillos <span>I’m a software develop
         </li>
       </ul>
 
-      <button class="btn-down">
+      <button type="button" class="btn-down">
         <img src="images/down-button.png" alt="button">
-
       </button>
+
     </section>
 
+    <section class="work">
+      <h2 class="title title--work">my recent work</h2>
+
+      <div class="card-container">
+        <article class="card">
+
+          <div class="card__imgbox">
+            <img src="images/work-screenshots/Bankist.PNG" alt="bankist-app" class="card__img">
+          </div>
+
+          <div class="card__details">
+            <h3 class="card__text">bankist website</h3>
+
+            <ul class="card__list">
+              <li>javaScript</li>
+              <li>css</li>
+              <li>React</li>
```

Comment:
- [ ] According to the requirement the project(card) should be` <li>` and inside a `<ul>` tag. 
- Your implementation using `articles` tags
```
 <div class="card-container">
      <article class="card"> </article>
      <article class="card"></article>
      <article class="card"></article>
      <article class="card"></article>
      <article class="card">
</div>
```
- A better implementation using `ul` tag

```
 <ul class="card-container">
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
</ul>
```